fastlane/metadata/android/en-US/changelogs/116.txt

```diff
@@ -0,0 +1,3 @@
+* Added data source for Czechia (Thanks Ondra!)
+* Display standard fares for trip results
+* Better error handling in case of failing trip queries
```

Comment:
Thanks for thinking of the changelog.

We should maybe document this and the entire release process somewhere. Normally, the changelog is first added to [`src/main/res/xml/changelog_master.xml`](https://github.com/grote/Transportr/blob/master/app/src/main/res/xml/changelog_master.xml) to appear in the UI after updates. Then, [`fastlane/generate_changelog.py`](https://github.com/grote/Transportr/blob/master/fastlane/generate_changelog.py) is run to produce the Google play changelog.

Reply:
I think the two lines above are missing a (Thanks Mikolai!) ;)

Reply:
Done :)